core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/ColumnFormatRelation.scala

```diff
@@ -322,16 +322,6 @@ abstract class BaseColumnFormatRelation(
 
       if (tableExists) {
         mode match {
-          case SaveMode.Ignore =>
-            dialect match {
-              case GemFireXDDialect =>
-                GemFireXDDialect.initializeTable(table,
-                  sqlContext.conf.caseSensitiveAnalysis, conn)
-                GemFireXDDialect.initializeTable(externalColumnTableName,
-                  sqlContext.conf.caseSensitiveAnalysis, conn)
-              case _ => // Do nothing
-            }
-            return
           case SaveMode.ErrorIfExists =>
             // sys.error(s"Table $table already exists.") TODO: Why so?
             return
```

Comment:
Ignore the above comment since initialization is done separately in createTable.